src/aics-image-viewer/components/TfEditor/index.tsx

```diff
@@ -739,66 +696,46 @@ export default class MyTfEditor extends React.Component<MyTfEditorProps, MyTfEdi
 
     return (
-      <div id="container">
-        <svg id={`svg-${id}`} width={width} height={height} ref={this.svgElement} />
-        <div className="aligned">
-          {this.state.displayColorPicker ? (
-            <div style={STYLES.popover}>
-              <div style={STYLES.cover} onClick={this.handleCloseColorPicker} />
-              <StatefulSketchPicker
-                color={colorArrayToObject(this.last_color)}
-                onChange={this.handleChangeColor}
-                disableAlpha={true}
-              />
-            </div>
-          ) : null}
+      <div>
+        <div className="button-row">
+          {this.createTFGeneratorButton("resetXF", "None", "Reset transfer function to full range.")}
+          {this.createTFGeneratorButton("auto98XF", "Default", "Ramp from 50th percentile to 98th.")}
+          {this.createTFGeneratorButton("auto2XF", "IJ Auto", `Emulates ImageJ's "auto" button.`)}
+          {this.createTFGeneratorButton("bestFitXF", "Auto 2", "Ramp over the middle 80% of data.")}
         </div>
-        <div className="aligned">
-          <Checkbox checked={colorizeEnabled} onChange={this.handleColorizeCheckbox} id={`colorize-${id}`}>
-            Colorize
-          </Checkbox>
-          <div style={STYLES.control}>
-            <Nouislider
-              //                      id={`svg-${id}`}
-              range={{ min: [0], max: [1] }}
-              start={colorizeAlpha}
-              connect={true}
-              tooltips={true}
-              behaviour="drag"
-              onUpdate={this.handleColorizeAlpha}
+        {colorPickerPosition !== null && (
+          <div style={{ ...STYLES.popover, ...{ [cpDirection]: Math.abs(colorPickerPosition) } }}>
+            <div style={STYLES.cover} onClick={this.handleCloseColorPicker} />
+            <StatefulSketchPicker
+              color={colorArrayToObject(this.last_color)}
+              onChange={this.handleChangeColor}
+              disableAlpha={true}
             />
           </div>
-        </div>
-        <div className="aligned">
-          <Button id={`reset-${id}`} className="ant-btn" onClick={this.resetXF}>
-            Reset
-          </Button>
-          <Button id={`auto-${id}`} className="ant-btn" onClick={this.autoXF}>
-            Auto
-          </Button>
-          <Button id={`bestfit-${id}`} className="ant-btn" onClick={this.bestFitXF}>
-            BestFit
-          </Button>
-          <Button id={`auto2-${id}`} className="ant-btn" onClick={this.auto2XF}>
-            Auto_IJ
-          </Button>
-          <Button id={`auto98-${id}`} className="ant-btn" onClick={this.auto98XF}>
-            Auto_98
-          </Button>
-        </div>
+        )}
+        <svg className="tf-editor-svg" width={width} height={height} ref={this.svgElement} />
+        <SliderRow
+          label={
+            <Checkbox checked={colorizeEnabled} onChange={(e) => this.props.updateColorizeMode(e.target.checked)}>
```

Comment:
It's slick that you can directly put a Checkbox into the label field here. Nice!